activity_browser/app/bwutils/metadata.py

```diff
@@ -102,5 +103,46 @@ def reset_metadata(self):
         self.dataframe = pd.DataFrame()
         self.databases = set()
 
+    def decompose_tuple_column(self, colname='categories'):
```

Comment:
Just make this function non-generic. It will only be used for unpacking the catagories tuples.
OR
Add parameter where a list of columnnames is passed and insert logic for generating columnnames if the list is empty.